Makefile

```diff
@@ -1,35 +1,61 @@
-.PHONY: get-code-generator generate build clean
+APP = tagger
+PLUGIN = kubectl-tag
 
-PROJECT=github.com/ricardomaraschini/tagger
-GEN_OUTPUT=/tmp/$(PROJECT)/infra/tags
+IMAGE_BUILDER ?= podman
+IMAGE ?= quay.io/rmarasch/tagger
+IMAGE_TAG = $(IMAGE):latest
 
-build:
-	go build -mod vendor -o _output/bin/tagger ./cmd/tagger
-	go build -mod vendor -o _output/bin/kubectl-tag ./cmd/kubectl-tag
+OUTPUT_DIR ?= _output
+OUTPUT_BIN = $(OUTPUT_DIR)/bin
 
+TAGGER_BIN = $(OUTPUT_BIN)/$(APP)
+PLUGIN_BIN = $(OUTPUT_BIN)/$(PLUGIN)
+GEN_BIN = $(OUTPUT_DIR)/code-generator
+
+PROJECT = github.com/ricardomaraschini/$(APP)
+GEN_OUTPUT = /tmp/$(PROJECT)/imagetags
```

Comment:
This is causing the issue we discussed. This variable has been changed from `/tmp/$(PROJECT)/infra/tags` to `/tmp/$(PROJECT)/imagetags` in this PR.

Reply:
Good catch! It should be fixed now.